.travis/ios/before_install.bash

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -eu
+
+# prepare android sdk
+
+if [[ ! -d "$ANDROID_HOME" ]]; then
+    pushd "$ANDROID_HOME"
+    curl -s# 'https://dl.google.com/android/repository/sdk-tools-darwin-4333796.zip' > android-sdk.zip
+    unzip android-sdk.zip
+    rm android-sdk.zip
+    popd
+fi
+
+brew tap caskroom/versions
+brew cask install java8
```

Comment:
the required macOS has Java10 but it fails our builds. So we need to install java8 manually.